NDTensors/test/NDTensorsTestUtils/device_list.jl

```diff
@@ -18,13 +19,21 @@ if "metal" in ARGS || "all" in ARGS
 end
 if "cutensor" in ARGS || "all" in ARGS
   Pkg.add("cuTENSOR")
+  Pkg.add("CUDA")
   using CUDA, cuTENSOR
```

Comment:
```suggestion
  using CUDA: CUDA
  using cuTENSOR: cuTENSOR
```

Reply:
Or maybe it should be:
```suggestion
  using cuTENSOR: cuTENSOR
  using CUDA: cu
```

Reply:
Or maybe it should be:
```suggestion
  using cuTENSOR: cuTENSOR
  using CUDA: cu
```

Reply:
I updated these and I bring in the `CUDA` name instead of `cu` because we use `NDTensors.cu` in the code not the opinionated (F32 preferred) `CUDA.cu` version. I am closing this comment. Thanks!